### DISCUSSION TITLE
Add region flag to disable AI from mobs, and another for turning off hoppers.

### DISCUSSION THREAD

**Discussion by nuttapillar** [GriefPrevention/GriefPrevention#2315] [Ideas]
Hi, I'd love a flag I can apply to regions that will disable the AI of any mob types I specify (or all) in the region. It'd be handy for reducing the server load on areas that don't need the mobs to run AI. I could chuck it on the bases of players who are away for a long time, so their base doesn't contribute to lag. And then when they return, I can lift the flag.

Perhaps additionally, a flag to turn off all hopper moving/checking in the region too. Could be very useful for optimisation.

**Comment by Jikoo** [comment]
Sounds like something you might want to request over at https://github.com/akdukaan/GPFlags

My 2 cents:
I'm assuming you mean passive or persistent entities and not mobs, because mobs shouldn't spawn unless there are players nearby, so you wouldn't save anything by disabling them.
"Turning off" hoppers sounds like a nightmare. You'd have to either listen to hopper events to stop them (which is exactly what you don't want) or force a redstone signal to the hopper, which could very easily break complex builds completely.

Maybe instead of all this you should just use WorldEdit to "archive" inactive bases by creating a schematic and deleting them. Player comes back, you can paste it back in, boom.
